src/scss/style.scss

```diff
@@ -39,3 +39,4 @@
 @import "block-patterns/link-list";
 @import "block-patterns/statistics";
 @import "block-patterns/page-header-breadcrumbs";
+@import "block-patterns/cards";
```

Comment:
Seems like this file should be called "card", not "cards" for consistency with the other block patterns.

Reply:
good idea. I named it cards because there just weren't enough styles to separate each card type into it's own partial so I'm doing all cards in there, including the global styles.